homeassistant/components/sense/sensor.py

```diff
@@ -130,26 +151,26 @@ class SenseActiveSensor(SensorEntity):
     _attr_extra_state_attributes = {ATTR_ATTRIBUTION: ATTRIBUTION}
     _attr_should_poll = False
     _attr_available = False
-    _attr_state_class = STATE_CLASS_MEASUREMENT
+    _attr_state_class = STATE_CLASS_TOTAL_INCREASING
```

Comment:
A sensor expressed in `Watt` is not a metered value and so is not total increasing